<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Header</class>
 <widget class="QWidget" name="Header">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}

QLabel
{
	color: white;
	font: 20px, &quot;微软雅黑&quot;;
}</string>
  </property>
  <widget class="QLabel" name="PageName">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>1</y>
     <width>72</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
font: 22px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Label1">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>1</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>操作模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DrivingMode">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>41</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
	font: 18px, &quot;微软雅黑&quot;;
color:yellow;</string>
   </property>
   <property name="text">
    <string>人工驾驶</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label2">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>1</y>
     <width>72</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>网压</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NetVoltage">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>41</y>
     <width>72</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
	font: 18px, &quot;微软雅黑&quot;;
color:yellow;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label3">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>1</y>
     <width>84</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Speed">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>41</y>
     <width>84</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
	font: 18px, &quot;微软雅黑&quot;;
color:yellow;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>1</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>级位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Level">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>41</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
	font: 16px, &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>1</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>当前站</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentStation">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>41</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 18px, &quot;微软雅黑&quot;;
color:yellow;
border: 2px solid rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label6">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>1</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>下一站</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NextStation">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>41</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 18px, &quot;微软雅黑&quot;;
color:yellow;border: 2px solid rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DateTime">
   <property name="geometry">
    <rect>
     <x>722</x>
     <y>1</y>
     <width>134</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="HelpItemBtn">
   <property name="geometry">
    <rect>
     <x>856</x>
     <y>1</y>
     <width>78</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/helpitem.bmp);
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CurrentFaultBtn">
   <property name="geometry">
    <rect>
     <x>934</x>
     <y>1</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border:2px solid black;
color: rgb(255, 0, 0);
font: 24px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>71</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-image: url(:/images/images/logo.bmp);
border:2px solid black;
color: rgb(255, 0, 0);
font: 24px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
